app/src/main/java/org/openimis/imispolicies/ClientAndroidInterface.java

```diff
@@ -6345,11 +6343,28 @@ public boolean getRule(String rulename) {
         return rule;
     }
 
+    @JavascriptInterface
+    public boolean CheckInternetAvailable() {
+        // check internet connection
+        General _General = new General(AppInformation.DomainInfo.getDomain());
+        if(!_General.isNetworkAvailable(mContext)) {
+            ShowDialog(mContext.getResources().getString(R.string.NoInternet));
+            return false;
+        }
+        return true;
+    }
+
     public void LoginDialogBox(final String page) {
 
         ((Activity) mContext).runOnUiThread(new Runnable() {
             @Override
             public void run() {
+                // check internet connection
+                General _General = new General(AppInformation.DomainInfo.getDomain());
```

Comment:
This is the same code as in CheckInternetAvailable method. Why you don't call the method?

Reply:
Changed to CheckInternetAvailable